web/taskmaster/src/app/app.component.ts

```diff
@@ -0,0 +1,36 @@
+import { Component, inject } from '@angular/core';
+import { RouterOutlet } from '@angular/router';
+import { MatListModule } from '@angular/material/list';
+import { TaskFormComponent } from './taskform.component';
+import { Task } from './task.model';
+import { TaskService } from './task.service';
+
+@Component({
+  selector: 'app-root',
+  standalone: true,
+  imports: [RouterOutlet, MatListModule, TaskFormComponent],
+  templateUrl: './app.component.html',
+  styleUrl: './app.component.scss'
+})
+export class AppComponent {
+  title = 'taskmaster';
+  tasks = Array<Task>();
+
+  taskService = inject(TaskService);
+
+  constructor() {
```

Comment:
The way `taskService = inject(TaskService)` is valid and useful for testing files and some things like pipes. The most common way to make use of dependency injection is with `constructor(private taskService: TaskService) {}`

Reply:
I was wondering about this. If I want to be able to use the service multiple times, should I assign it to a property like 
```typescript
constructor(private taskService: TaskService) {
  this.taskService = taskService
}
```
??

Reply:
> I was wondering about this. If I want to be able to use the service multiple times, should I assign it to a property like
> 
> ```ts
> constructor(private taskService: TaskService) {
>   this.taskService = taskService
> }
> ```
> 
> ??

Looks like is not required

Reply:
When you inject the service in the constructor you don't need to assign a new variable for it. Using `private/public` within the constructor allows you to access the service as `this.taskService` directly from the injection as if there is a variable available already.